### DISCUSSION TITLE
Emulator crashes when loading any rom on Windows 10

### DISCUSSION THREAD

**Discussion by AlexN64gamer** [project64/project64#2004] [Support]
Hi, im using Project64 2.4.0.1552 dev with the following plugins: angrylions rdp plus 1.6 dirty, shunyuans HLE audio 1.8.2, Nrage (default plugin) and RSP (default plugin), the issue is when trying to load any rom, the emulator freezes on "not responding" for a long time before starting the rom

**Comment by flagrama** [comment]
Does the same issue occur if you use all default plugins?

**Comment by Squall-Leonhart** [comment]
likely caused by 7z unpacking into memory.
